lifterlms.php

```diff
@@ -5,12 +5,12 @@
  * @package LifterLMS/Main
  *
  * @since 1.0.0
- * @version 3.37.0
+ * @version 3.38.0-beta.1
```

Comment:
not sure references to the beta here are correct, we probably should use the version placeholder?!